### ISSUE TITLE
Perhaps change "da_DK" to "da"

### ISSUE THREAD

**Issue by scootergrisen** [lxde/lxqt#1203]
For danish translation i see both da and da_DK files.

Example:
https://github.com/lxde/lxqt-l10n/tree/master/lxqt-panel/plugin-clock

The da_DK seems to be more translated then the da in the example.
But i would prefer using da when there are no other da_* translations.

**Comment by pmattern** [comment]
This should definitely be done, yes. For some background information see section [Language vs. country specific translations](https://github.com/lxde/lxqt/wiki/Translation#language-vs-country-specific-translations) in the wiki page addressing translation.

Any chance you can submit corresponding PRs?

**Comment by scootergrisen** [comment]
I just learned that LXQt exists the other day so i haven't translated anything directly for it yet so maybe better to ask the people that have worked on so i don't do something wrong.
But i can try checking the files to see how many da/da_DK there are for at start.

**Comment by pmattern** [comment]
Alright. The wiki page mentioned above states which kinds of localization files exist and where to find them, too. So if you'd like to help out, which is always much appreciated, you may want to look through those files and let us know what to do with them, e. g. merge a country- into the language-specific one, drop a country-specific one altogether or whatever.

**Comment by scootergrisen** [comment]
Qt Linguist defaults to selecting both the language and country.
So if i open a not yet translated TS file with no language attribute in the TS tag, Linguist will defaults to Danish and Denmark.

So people would have to select "Any Country" for each new file to switch to just "da".
There don't seem to be a setting for prefering "Any Country" for files without language attributes but i guess i could suggest it to Qt.

That might explain why some are "da" and some are "da_DK" and some have both files.

**Comment by pmattern** [comment]
True. If a TS file is translated the first time and Qt Linguist is used it's necessary to set "Any Country" to avoid country-specific translations. And yes, translators missing this may be the reason why language- and country-specific translations happen to exist in parallel where it does not make sense.

Not sure whether this is really relevant for the topic we're discussing here, namely what to do when a language- and country-specific translation of a particular file are already preexisting in parallel, though.
Basically I think tools like `diff` or `sdiff` are the better choice here. E. g. in directory `./liblxqt` of [lxqt-l10n](https://github.com/lxde/lxqt-l10n) run `sdiff liblxqt_da.ts liblxqt_da_DK.ts | less`, see whether the latter comprises translations missing in the former, transfer those translations and drop the country-specific files.

**Comment by scootergrisen** [comment]
I think i have managed to change all da_DK to da.
If anyone would like to double check for me if "da_DK" exists anywhere that would be nice.
Otherwise this issue can be closed.
